ui/widgets/nuxeo-selectivity.js

```diff
@@ -6802,7 +6802,7 @@ typedArrayTags[weakMapTag] = false;
 */
 
 .selectivity-dropdown {
-  background: #fff;
+  background: var(--nuxeo-dropdown-list-background);
```

Comment:
We should also add a fallback in case the variable is not present (e.g. renamed, not defined, theme was not loaded, etc.).

Something like `var(--nuxeo-dropdown-list-background, #fff);`

Reply:
Same applies to the other variables ;)